packages/beacon-node/test/unit/network/subscribeBoundary.test.ts

```diff
@@ -0,0 +1,232 @@
+    electra: 10,
+    fulu: 20,
+    blobSchedule: [
+      {EPOCH: 30, MAX_BLOBS_PER_BLOCK: 300},
+      {EPOCH: 40, MAX_BLOBS_PER_BLOCK: 400},
+    ],
+    testCases: [
+      {
+        epoch: 20,
+        activeBoundaries: [
+          {fork: ForkName.electra},
+          {fork: ForkName.fulu, EPOCH: 10, MAX_BLOBS_PER_BLOCK: defaultConfig.MAX_BLOBS_PER_BLOCK_ELECTRA},
+        ],
+      },
+      {
+        epoch: 25,
+        activeBoundaries: [
+          {fork: ForkName.fulu, EPOCH: 10, MAX_BLOBS_PER_BLOCK: defaultConfig.MAX_BLOBS_PER_BLOCK_ELECTRA},
+        ],
+      },
+      {
+        epoch: 30,
+        activeBoundaries: [
+          {fork: ForkName.fulu, EPOCH: 10, MAX_BLOBS_PER_BLOCK: defaultConfig.MAX_BLOBS_PER_BLOCK_ELECTRA},
+          {fork: ForkName.fulu, EPOCH: 30, MAX_BLOBS_PER_BLOCK: 300},
+        ],
+      },
+      {epoch: 35, activeBoundaries: [{fork: ForkName.fulu, EPOCH: 30, MAX_BLOBS_PER_BLOCK: 300}]},
+      {
+        epoch: 40,
+        activeBoundaries: [
+          {fork: ForkName.fulu, EPOCH: 30, MAX_BLOBS_PER_BLOCK: 300},
+          {fork: ForkName.fulu, EPOCH: 40, MAX_BLOBS_PER_BLOCK: 400},
+        ],
+      },
+      {epoch: 45, activeBoundaries: [{fork: ForkName.fulu, EPOCH: 40, MAX_BLOBS_PER_BLOCK: 400}]},
+    ],
+  },
+  {
+    altair: 0,
+    bellatrix: 0,
+    capella: 0,
+    deneb: 0,
+    electra: 10,
+    fulu: 20,
+    blobSchedule: [
+      {EPOCH: 22, MAX_BLOBS_PER_BLOCK: 220},
+      {EPOCH: 24, MAX_BLOBS_PER_BLOCK: 240},
+    ],
+    testCases: [
+      {
+        epoch: 20,
+        activeBoundaries: [
+          {fork: ForkName.electra},
+          {fork: ForkName.fulu, EPOCH: 10, MAX_BLOBS_PER_BLOCK: defaultConfig.MAX_BLOBS_PER_BLOCK_ELECTRA},
+          {fork: ForkName.fulu, EPOCH: 22, MAX_BLOBS_PER_BLOCK: 220},
+        ],
+      },
+      {
+        epoch: 23,
+        activeBoundaries: [
+          {fork: ForkName.fulu, EPOCH: 10, MAX_BLOBS_PER_BLOCK: defaultConfig.MAX_BLOBS_PER_BLOCK_ELECTRA},
+          {fork: ForkName.fulu, EPOCH: 22, MAX_BLOBS_PER_BLOCK: 220},
+          {fork: ForkName.fulu, EPOCH: 24, MAX_BLOBS_PER_BLOCK: 240},
+        ],
+      },
+      {
+        epoch: 25,
+        activeBoundaries: [
+          {fork: ForkName.fulu, EPOCH: 22, MAX_BLOBS_PER_BLOCK: 220},
+          {fork: ForkName.fulu, EPOCH: 24, MAX_BLOBS_PER_BLOCK: 240},
+        ],
+      },
+      {epoch: 27, activeBoundaries: [{fork: ForkName.fulu, EPOCH: 24, MAX_BLOBS_PER_BLOCK: 240}]},
+    ],
+  },
+];
+
+for (const testScenario of testScenarios) {
+  const {altair, bellatrix, capella, deneb, electra, fulu, blobSchedule, testCases} = testScenario;
+
+  describe("network", () => {
+    const forkConfig = getForkConfig({altair, bellatrix, capella, deneb, electra, fulu, blobSchedule});
+    for (const testCase of testCases) {
+      const {epoch, activeBoundaries} = testCase;
+      it(` on epoch ${epoch} should return getActiveSubscribeBoundaries: ${JSON.stringify(activeBoundaries)}`, () => {
```

Comment:
test output is pretty verbose already, could make it a bit less repetitive

```suggestion
  describe("network / forks / getActiveSubscribeBoundaries", () => {
    const forkConfig = getForkConfig({altair, bellatrix, capella, deneb, electra, fulu, blobSchedule});
    for (const testCase of testCases) {
      const {epoch, activeBoundaries} = testCase;
      it(` on epoch ${epoch} should return ${JSON.stringify(activeBoundaries)}`, () => {
```